docs/articles/posts/20250529_adam_hollings_interview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Interview with a Data Scientist - Dimsum and Datascience
+authors: [adamhollings]
```

Comment:
your author ID thing needs to be identical to the one in the authors.yml or it breaks the website
```suggestion
authors: [AdamHollings]
```